light-service.gemspec

```diff
@@ -18,6 +18,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 2.6.0'
 
   gem.add_runtime_dependency("activesupport", ">= 4.0.0")
+  gem.add_runtime_dependency("i18n")
+  gem.add_runtime_dependency("dry-inflector")
+  gem.add_runtime_dependency("structured_warnings")
```

Comment:
Oh, I missed these additions. So instead of having just AS, we are adding 3 more dependencies.
I wish we could achieve the same functionality with no additional gems. But maybe I am just asking too much.

I wonder what ppl think about this change.

Reply:
We’re adding only 2 more dependencies, since i18n is already in through AS at the moment.

I explicitly required it as a dep in order to have a functioning gemspec even considering AS removal and without leaving remainders for future work.

Speaking about the balance between AS and the two new gems: my sentiment is that with the two gems we bring less code than with AS alone. We should analyze it more scientifically tho.

For sure we have the achievement to not have a big rails ecosystem’s gem, leaving LS more desirable as cross-framework and franework-less solution.

Speaking of deprecation warnings: I previously tried to implement them using Gem::Deprecate, but it resulted to me as an inadequate tool out of its absolute standard concept. Hard to adapt. I think it would be feasible to write a dedicated Logger and implement some logic inside it in order to achieve a simple custom deprecation warner. This would be intended as a pragmatic and simpler alternative to structured_warning gem.
If you’d like to check a draft implementation I could try to write a separate draft PR in the second half of July.

About the inflector I’m really opinionated: I would not even try to reinvent the wheel on such a matter. Moreover I sincerely  invite you to read dry-inflector’s source code: it’s astonishing small, tidy and simple; and it’s a completely standalone gem. Obviously we could copy that few methods we need, but pro and cons considered I think it’s a solid choice to bring it in.

Thanks for the feedback 😊